test/e2e/framework/kubernetes/install-retina-helm.go

```diff
@@ -42,6 +42,16 @@ func (i *InstallHelmChart) Run() error {
 
 	tag := os.Getenv(i.TagEnv)
 
+	imageRegistry := os.Getenv("IMAGE_REGISTRY")
```

Comment:
Mat can comment on this more, but given it's a go code, I would prefer test flags than environment variables. This way we can validate immediately if correct flags were passed or not.

Reply:
No strong opinion, but I don't consider image registry to be a value that's frequently changed, so for those type of things I usually stick to env's, as it's easier to add to some env.json or .bashrc and forget about it. Other things that impact output or behavior of the test, yeah flags for sure

Reply:
Image registry can have some default values and go test help can show the default values and validate stuff at the very start.

Reply:
It is changed into flags, for image-registry, image-namespace and tag provided by the user will provide those. Help or not providing the value will bring up the doc on expected flags.